peachjam/js/components/Annotations/Annotation.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <la-gutter-item
+    :id="annotation.id"
+    :anchor.prop="anchorElement"
+  >
+    <div class="card">
+      <div class="card-body">
+        <div>
+          <div class="dropdown text-end">
+            <a
+              id="dropdownMenuButton"
+              class="btn btn-link"
+              type="button"
+              data-bs-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="bi bi-three-dots" />
+            </a>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              <a class="dropdown-item" role="button" @click="editAnnotation">Edit</a>
+              <a class="dropdown-item" role="button" @click="deleteAnnotation">Delete</a>
+            </div>
+          </div>
+        </div>
+        <textarea
+          v-if="editing"
+          v-model="annotation.text"
+          class="form-control grow-text"
+        />
+        <p v-else>
+          {{ annotation.text }}
+        </p>
+        <div class="mt-2 text-end">
+          <button
+            v-if="editing"
+            class="ms-1 btn btn-sm btn-secondary"
+            @click="editing = false"
+          >
+            Cancel
```

Comment:
if the annotation is new, emit a delete event so that the list can delete it completely.

Reply:
done, we now emit an event to remove the annotation fromt the list